packages/transaction-controller/src/gas-flows/OptimismLayer1GasFeeFlow.test.ts

```diff
@@ -66,10 +70,18 @@ describe('OptimismLayer1GasFeeFlow', () => {
 
   describe('getLayer1GasFee', () => {
     it('returns layer 1 gas fee', async () => {
+      const txFactorySpy = jest.spyOn(TransactionFactory, 'fromTxData');
       const flow = new OptimismLayer1GasFeeFlow();
       const response = await flow.getLayer1Fee(request);
 
       expect(contractGetL1FeeMock).toHaveBeenCalledTimes(1);
+      expect(txFactorySpy).toHaveBeenCalledWith(
```

Comment:
Not a blocker but I was meaning we can assert the contents of the call to `contractGetL1Fee`.